rskj-core/src/test/java/co/rsk/peg/BridgeSupportTest.java

```diff
@@ -458,8 +458,12 @@ void getActiveFederatorPublicKeyOfType() {
 
         @Test
         void getRetiringFederation() {
-            when(federationSupport.getRetiringFederation()).thenReturn(federation);
-            assertThat(bridgeSupport.getRetiringFederation(), is(federation));
+            Optional<Federation> expectedRetiringFederation = Optional.of(federation);
+            when(federationSupport.getRetiringFederation()).thenReturn(expectedRetiringFederation);
```

Comment:
```suggestion
            when(federationSupport.getRetiringFederation()).thenReturn(Optional.of(federation));
```
same here, i think this way is clearer. What do you think?